src/kit/List.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ReactNode } from 'react';
+
+import Button from 'kit/Button';
+import Column from 'kit/Column';
+import Dropdown from 'kit/Dropdown';
+import Icon, { IconName } from 'kit/Icon';
+import { ElevationWrapper } from 'kit/internal/Elevation';
+import { AnyMouseEventHandler } from 'kit/internal/types';
+import Row from 'kit/Row';
+
+import css from './List.module.scss';
+
+interface List {
+  items: ListItem[];
+}
+
+interface Action {
+  name: string;
+  onClick: () => void;
+}
+
+interface ListColumn {
+  width?: number;
+  content: ReactNode;
+}
+
+export interface ListItem {
+  icon: IconName;
+  title: string;
+  subtitle?: ReactNode;
+  buttons?: Action[];
+  menu?: Action[];
+  columns?: ListColumn[];
+  onClick?: AnyMouseEventHandler;
+}
+
+const getMenuOptions = (menu?: Action[]) => {
+  return menu?.map((m, idx) => {
+    return {
+      key: idx,
+      label: m.name,
+    };
+  });
+};
+
+const List: React.FC<List> = ({ items }: List) => {
+  return (
+    <div>
+      {items.map((row, idx) => {
+        const rowClasses = [css.item];
+        if (row.onClick) rowClasses.push(css.clickable);
+        if (row.subtitle) rowClasses.push(css.hasSubtitle);
+
+        return (
+          <ElevationWrapper className={rowClasses.join(' ')} key={idx} onClick={row.onClick}>
+            <Row height={row.subtitle ? 52 : 40}>
+              <Row width="fill">
+                <span className={css.icon}>
+                  <Icon decorative name={row.icon} size="small" />
+                </span>
+                <Row justifyContent="space-between" width="fill">
+                  <Column gap={0} width="hug">
+                    <strong>{row.title}</strong>
+                    {row.subtitle && <span className={css.subtitle}>{row.subtitle}</span>}
+                  </Column>
+                  {row.columns?.map((col, idx) => {
+                    return (
+                      <Column key={idx} width={col.width ? col.width : 'hug'}>
+                        {col.content}
+                      </Column>
+                    );
+                  })}
+                </Row>
+              </Row>
+            </Row>
+            {((row.buttons && row.buttons.length > 0) || (row.menu && row.menu.length > 0)) && (
+              <div
+                className={css.actions}
+                onClick={(e) => {
+                  e.stopPropagation();
+                }}>
+                <Row>
+                  {row.buttons?.map((b, idx) => {
```

Comment:
I think in grooming we said this specifically should be a manually defined list of actions for buttons/menu instead of having actions move between them responsively.

The mobile issue is a good point, my guess is that these actions are generally things that can also be done from clicking through the row to another page, and surfacing them here is a convenience that's ok to not have on mobile.

Don't think I'm reproducing the button overflow issue you're seeing.